City-of-Helsinki/berth-reservations
# VEN-933 | Add missing ProductServiceType

## Description :sparkles:
*  Add missing ProductServiceType (`Storage on ice`) and translations

## Issues :bug:
### Closes :no_good_woman:
**[VEN-933](https://helsinkisolutionoffice.atlassian.net/browse/VEN-933):** Add missing product services

### Related 🤝 
**[VEN-814](https://helsinkisolutionoffice.atlassian.net/browse/VEN-814):** Pricing page: support "Add New" additional services

## Testing :alembic:
### Automated tests :gear:️
N/A

### Manual testing :construction_worker_man:
1. Go to add a new `AdditionalProduct`
http://localhost:8000/admin/payments/additionalproduct/add/
2. You should see the `Storage on ice` option

## Screenshots :camera_flash:
<img width="363" alt="image" src="https://user-images.githubusercontent.com/15201480/96585574-7768c680-12e8-11eb-9e3a-733c04471f1e.png">
